servant-server/test/Servant/Server/UsingConfigSpec/TestCombinators.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- | These are custom combinators for S.S.UsingConfigSpec. It's split up into
+-- its own module to be able to test how exactly module import work when using
+-- the config.
```

Comment:
I still think this comment isn't very helpful in understanding what part of module import is being tested.